platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -190,14 +205,15 @@ public MigrationReport doDecrypt(Path exportDirectory, Consumer<MigrationMessage
   }
 
   @VisibleForTesting
-  void delegateToImportMigrationManager(MigrationReportImpl report, MigrationZipFile zip) {
+  void delegateToImportMigrationManager(
+      MigrationReportImpl report, MigrationZipFile zip, Set<String> mandatoryMigratables) {
     final ImportMigrationManagerImpl mgr =
-        new ImportMigrationManagerImpl(report, zip, migratables.stream());
+        new ImportMigrationManagerImpl(report, zip, mandatoryMigratables, migratables.stream());
     try {
       report.record(Messages.IMPORTING_DATA, productBranding, zip.getZipPath());
       mgr.doImport(productBranding, productVersion);
     } finally {
-      IOUtils.closeQuietly(mgr);
+      IOUtils.closeQuietly(mgr); // do not care if we fail to close the mgr/zip file!!!
```

Comment:
I think by convention we've avoided `!` in comments or logging statements

Reply:
But they were so nice to make my point 😄 
Removed 😞  😄